packages/compass-sidebar/src/components/multiple-connections/header/sidebar-header.tsx

```diff
@@ -29,7 +29,7 @@ export function SidebarHeader({
   onAction(actionName: Action): void;
 }): React.ReactElement {
   return (
-    <div className={sidebarHeaderStyles}>
+    <div className={sidebarHeaderStyles} data-testid="sidebar-header">
```

Comment:
testid used in e2e selectors for some custom browser commands that need to go click somewhere/anywhere that will have no effect other than that it will blur whatever is actually focused and give some browser.waitUntil() a chance to succeed next time.

Total hack. Documented in the custom commands but that doesn't necessarily show up in the diff in this PR so I figured I'd just leave this comment here.